science/lalinference/Portfile

```diff
@@ -3,17 +3,16 @@
 PortSystem    1.0
 
 name          lalinference
-version       1.10.1
-revision      0
+version       2.0.3
```

Comment:
Please retain `revision 0` lines and add them anywhere they're missing. We're switching to always listing the revision, even if it's 0, because it makes it easier to see where to increase ports' revisions, especially in ports with subports.

It's ok to update everything at once this time, but it's probably better in the future to use a separate commit for each port being updated, even if the several commits are submitted in a single PR.

Could you elaborate the commit message a little? For example, you might add a line for each port, e.g.
```
lalinference: Update to 2.0.3
lalinspiral: Update to 1.10.1
```

etc.

Reply:
Thanks! Both done.